### PR TITLE
Add f(x)Core chain and FX token

### PR DESCRIPTION
**Pull request by devon-chain**
## What is the purpose of the change
To add the f(x)Core chain and FX token.

## Brief Changelog

add fx.svg (token logo),
add f(x)Core chain and FX token to source-chain-config.ts
add FX token to ibc-assets.ts

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

## Pending

* Pool ID for price definition
* Validation

### CODE REVIEW

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -1999,6 +1999,14 @@ export const IBCAssetInfos: (IBCAsset & {
         coinMinimalDenom: "uqsomm",
         isVerified: true,
       },
+      {
+        counterpartyChainId: "cosmoshub-4",
+        sourceChannelId: "channel-4",
+        destChannelId: "channel-141",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:2005]: this doesn't look right to me. Please register the IBC connection to the Chain Registry.

Comment (devon-chain) [reply packages/web/config/ibc-assets.ts:2005]: Hi @JeremyParish69 , FX token will cross the chain to osmosis via cosmoshub, so it is registered as a channel and ibc token on cosmoshub

File: packages/web/config/ibc-assets.ts [hunk 2]
```diff
@@ -1999,6 +1999,14 @@ export const IBCAssetInfos: (IBCAsset & {
         coinMinimalDenom: "uqsomm",
         isVerified: true,
       },
+      {
+        counterpartyChainId: "cosmoshub-4",
+        sourceChannelId: "channel-4",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:2004]: channel-0

File: packages/web/config/generate-chain-infos/source-chain-infos.ts [hunk 3]
```diff
@@ -3990,6 +3998,34 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
...
+    chainName: "f(x)Core",
+    bip44: {
+      coinType: 60,
+    },
+    bech32Config: Bech32Address.defaultBech32Config("fx"),
+    currencies: [
+      {
+        coinDenom: "FX",
+        coinMinimalDenom: "FX",
+        coinDecimals: 18,
```
Comment (JeremyParish69) [inline packages/web/config/generate-chain-infos/source-chain-infos.ts:4014]: As per the Chain Registry, the decimals should be 0--meaning FX is indivisible.
https://github.com/cosmos/chain-registry/blob/master/fxcore/assetlist.json

Comment (LanceLan96) [reply packages/web/config/generate-chain-infos/source-chain-infos.ts:4014]: resolved

File: packages/web/config/ibc-assets.ts (hunk 2 above)
Comment (LanceLan96) [reply packages/web/config/ibc-assets.ts]: resolved

**Comment by LanceLan96** [comment]
changed to direct channel between fxcore and osmosis

**Comment by JeremyParish69** [comment]
Replaced by https://github.com/osmosis-labs/osmosis-frontend/pull/2113

**Pull request closed without merging**